mysql-persistence/src/main/java/com/netflix/conductor/dao/mysql/MySQLExecutionDAO.java

```diff
@@ -174,7 +174,7 @@ public void removeTask(String taskId) {
 			return;
 		}
 
-		String taskKey = task.getReferenceTaskName() + "" + task.getRetryCount();
+		String taskKey = task.getReferenceTaskName() + "_" + task.getRetryCount();
```

Comment:
@grfeng seems this will also break existing data stored in the database.

Reply:
@v1r3n Look this https://github.com/grfeng/conductor/blob/a3194665493238d6e118e34ca5b1df702942e53c/mysql-persistence/src/main/java/com/netflix/conductor/dao/mysql/MySQLExecutionDAO.java#L101